hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/shell/volume/CreateVolumeHandler.java

```diff
@@ -58,9 +62,12 @@ protected void execute(OzoneClient client, OzoneAddress address)
     VolumeArgs.Builder volumeArgsBuilder = VolumeArgs.newBuilder()
         .setAdmin(adminName)
         .setOwner(ownerName);
-    if (quota != null) {
-      volumeArgsBuilder.setQuota(quota);
+    if (storagespaceQuota!= null) {
+      volumeArgsBuilder.setStoragespaceQuota(storagespaceQuota);
     }
+
+    volumeArgsBuilder.setNamespaceQuotaQuota(namespaceQuota);
```

Comment:
The function name seems a little redundant.

Suggestion is commented on `VolumeArgs.java`